content/docs/getting-started/installation/helm/modules/observability/_index.md

```diff
@@ -115,16 +115,14 @@ The Grafana dashboards require Grafana to be deployed in the same Kubernetes clu
 | ----------------- | --------------------------------------------------------- |
 | 10.x      | [Grafana Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana) |
 
-**Note:** From Grafana 10.x, deprecation warnings for Angular plugins will appear in the UI, but dashboards still work. **Grafana 11.x isn’t supported yet**
+**Note:** From Grafana 10.x, deprecation warnings for Angular plugins will appear in the UI, but dashboards still work. **Use of Grafana 11.x is recommended**
```

Comment:
Can we remove this note completely as it was added last release mentioning observability module won't work with 11.x. Now as we have added the support for 11.x, it would be good to remove it or modifying it to only mention that Grafana 11.x is required otherwise, Grafana will show deprecation warnings with below versions.

Reply:
Removed the note